extensions/applicationinsights-debugplugin-js/README.md

```diff
@@ -0,0 +1,188 @@
+# Microsoft Application Insights JavaScript SDK - Debug Plugin
+
+[![Build Status](https://travis-ci.org/microsoft/ApplicationInsights-JS.svg?branch=master)](https://travis-ci.org/microsoft/ApplicationInsights-JS)
+[![npm version](https://badge.fury.io/js/%40microsoft%2Fapplicationinsights-debugplugin-js.svg)](https://badge.fury.io/js/%40microsoft%2Fapplicationinsights-debugplugin-js)
+<!-- [![gzip size](https://img.badgesize.io/https://az416426.vo.msecnd.net/scripts/b/ai.2.min.js.svg?compression=gzip&softmax=30000&max=35000)](https://img.badgesize.io/https://az416426.vo.msecnd.net/scripts/b/ai.2.min.js.svg?compression=gzip&softmax=30000&max=35000) -->
+<!-- [![minified size size](https://img.badgesize.io/https://az416426.vo.msecnd.net/scripts/b/ai.2.min.js.svg?label=minified%20size)](https://img.badgesize.io/https://az416426.vo.msecnd.net/scripts/b/ai.2.min.js.svg?label=minified%20size) -->
+
+
+This README is broken down into several parts:
+
+- **Supported Browsers**
+- **Who** should use this plugin
+- **What** this plugin does
+- **When** to use this plugin
+- **Where** to get more information
+- **Why** this plugin exists
+- **Getting Started**
+
+these parts go together to explain the reasoning for this plugin's existance and provides
+an explaination as to how everything works. Each of these sections will finish with a
+bullet-point list that summarizes the section
+
+## Supported Browsers
+
+Our goal for the Application Insights Javascript SDK is to at least ensure that older, browsers
+running ES3 (IE7) don't crash when this plugin is loaded. **However,** because this is a
+*UI-based* plugin tailored for **developers** that is **not** meant to be used in a finalized
+website, there is an expectation that developers would be testing and debugging this particular
+data in modern browsers with modern tooling.
+
+As such, supporting Internet Explorer 7 and other browsers running ES3 is no-longer a priority
+for this plugin.
+
+This can be expected to work in:
+
+- Firefox
+- Chrome
+- Edge (new and old)
+
```

Comment:
We should include IE, we should be able to support IE9+ as they are ES5 or greater, but at the very least it should work in IE11

Reply:
I'm trying to work in support for IE11 in the inspector view, but that might take quite a bit of refactoring due to the way appendchild seems to refuse to work properly in IE11. It doesn't crash the page/browser though, so if that's what we're going for then it's fine.